### DISCUSSION TITLE
i can not install kimail2-1.143 on xampp

### DISCUSSION THREAD

**Discussion by ahmedgomaa** [kimai/kimai#2723] [On-Premise: Installation & Updates]
dear all
i download  kimail2-1.14 on C:\xampp\htdocs 
and nothing is happen
![image](https://user-images.githubusercontent.com/5402196/130319531-1b111af2-b196-488f-ad91-74ba93684369.png)
please help me urgently

**Comment by pkaltenboeck** [comment]
Your webserver isn't configured correctly, the DocumentRoot has to point to the public directory, otherwise the paths are not correct.

See https://www.kimai.org/documentation/webserver-configuration.html for more details.

**Reply by ahmedgomaa** [reply]
i have a xampp webserver and i can not install kimail2 on it
please help me in details

**Reply by ahmedgomaa** [reply]
![image](https://user-images.githubusercontent.com/5402196/132089169-88df44d9-21f3-4a70-ba30-b7e152f44c69.png)
i add the code in .htaccess in folder public
and still i can not install

**Comment by ahmedgomaa** [comment]
![image](https://user-images.githubusercontent.com/5402196/132089210-8b674a63-8a94-456f-a129-2e591ec7a5d2.png)
when i write 127.0.0.1/kimai2/public

**Reply by kevinpapst** [reply]
https://www.kimai.org/documentation/faq.html#i-have-a-problem

**Reply by ahmedgomaa** [reply]
؟؟؟
i can not find 
Please check your logfile at **var/log/prod.log.**

**Reply by ahmedgomaa** [reply]
please any help urgently

**Reply by kevinpapst** [reply]
You will not get any help without posting details from your log files and what you already tried to achieve.
And using words like `urgently` will only lead to users unsubscribing from your discussion.

Sorry that it doesn't work for you, but this forum is not here to teach you to webserver knowledge. Please ask at Stackoverflow or similar sides.

**Reply by ahmedgomaa** [reply]
thx alot for your help
and i try to install and i add my req. but i can not do
https://github.com/kevinpapst/kimai2/discussions/2723#discussioncomment-1383247
if you can help me
